languages/tree-sitter-stack-graphs-java/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "tree-sitter-stack-graphs-java"
+version = "0.1.0"
+description = "Stack graphs for precise code navigation for the Java programming language"
+
+homepage = "https://github.com/github/stack-graphs/tree/main/stack-graphs/languages/tree-sitter-stack-graphs-java"
+repository = "https://github.com/github/stack-graphs"
+license = "MIT OR Apache-2.0"
+readme = "stack-graphs-java/README.md"
```

Comment:
This path doesn't seem right.

Reply:
That is a remnant from the previous folder structure I was using - I thought I caught all instances but apparently not! Thank you for catching this. Pushing up a fix 🙇🏻